.github/workflows/examples-esp32.yaml

```diff
@@ -51,8 +51,8 @@ jobs:
             - name: Copy aside build products
               run: |
                   mkdir -p example_binaries/$BUILD_TYPE-build
-                  cp examples/wifi-echo/server/esp32/build/chip-wifi-echo.elf \
-                     example_binaries/$BUILD_TYPE-build/chip-wifi-echo.elf
+                  cp examples/all-clusters-app/esp32/build/chip-all-clusters-app.elf \
```

Comment:
supernit: given that names are long, maybe we could drop 'app' suffix and have 'all-clusters'.

Reply:
To be honest I don't have a very strong opinion about the name. 

I've added '-app' mostly because we already have 'lighting-app' and 'lock-app' and I was thinking that, maybe, the '-app' suffix may help folks to realise which folders are apps.

But if someone has a real strong opinion I will happily change that.